editor/plugins/org.fusesource.ide.wsdl2rest.ui/src/org/fusesource/ide/wsdl2rest/ui/internal/i18n/messages.properties

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2017 Red Hat, Inc.
```

Comment:
shouldn't the path be .../l10n/messages.properties?

Reply:
Heh. Oops!